stream-chat-android-client-sample/src/main/java/io/getstream/chat/android/client/sample/DocumentationSamples.kt

```diff
@@ -479,7 +480,7 @@ fun queryChannels() {
         .put("type", "messaging")
     val offset = 0
     val limit = 10
-    val sort = QuerySort().desc("last_message_at")
+    val sort = QuerySort.desc(Channel::lastMessageAt)
```

Comment:
ok, but how do we generate the API call to the backend using this setup?

Reply:
As previously. Nothing changed. There is toDto method for that purpose

Reply:
toDto?

Reply:
Yes, `QuerySort::toDto`